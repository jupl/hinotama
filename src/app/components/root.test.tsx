import {Provider} from 'mobx-react'
import * as React from 'react'
import {create} from 'react-test-renderer'
import Root from './root'

describe('<App.Root> Template', () => {
  it('should render as expected', () => {
    const stores = {}
    const component = create((
      <Provider {...stores}>
        <Root />
      </Provider>
    ))
    expect(component.toJSON()).toMatchSnapshot()
  })
})
