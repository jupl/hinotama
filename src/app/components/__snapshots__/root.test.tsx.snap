exports[`<App.Root> Template should render as expected 1`] = `
<div
  className="hSksZZ">
  Hello, World
</div>
`;
